docker/install/ubuntu_install_cmake_source.sh

```diff
@@ -20,8 +20,8 @@ set -e
 set -u
 set -o pipefail
 
-v=3.14
-version=3.14.7
+v=3.22
```

Comment:
Do we need to update the minimum cmake version in the cmake file?

Reply:
No because the update is only required by tflite build, not TVM

Reply:
does tflite require 3.22? #12682 is also trying to update the CMake version to 3.18 and we should pick the lower of the two for what runs in CI (if possible)

Reply:
3.18 is also accepted by tflite. Will do the change here and notify.